source/viz.md

```diff
@@ -1031,14 +1099,15 @@ the `sort_values` function followed by the `iloc` property.  Then to help us mak
 space, we'll use horizontal bars instead of vertical ones. We do this by
 swapping the `x` and `y` variables:
 
-```{index} pandas.DataFrame; sort_values, pandas.DataFrame; iloc[]
+```{index} pandas.DataFrame; nlargest
 ```
 
 ```{code-cell} ipython3
-islands_top12 = islands_df.sort_values(by = "size", ascending=False).iloc[:12] 
+islands_top12 = islands_df.nlargest(12)
 
 islands_bar_sorted = alt.Chart(islands_top12).mark_bar().encode(
```

Comment:
This code doesn't work -- stack trace printed

Reply:
resolved with 1104